### PR TITLE
Limited Stripe price lookups

### PR DESCRIPTION
**Pull request by SimonBackx**
refs https://github.com/TryGhost/Team/issues/2262

Makes sure we only loop active Stripe prices. If we find an inactive price, we also update it in our database now after this change.

### CODE REVIEW

File: ghost/payments/lib/payments.js [hunk 1]
```diff
@@ -214,8 +214,9 @@ class PaymentsService {
             stripe_product_id: product.id,
             currency,
             interval: cadence,
-            amount
-        }).query().select('stripe_price_id');
+            amount,
+            active: 1
```
Comment (rishabhgrg) [inline ghost/payments/lib/payments.js:218]: @SimonBackx Gotten a bit confused here but should this be `true` or `1` here ?

Comment (SimonBackx) [reply ghost/payments/lib/payments.js:218]: Wasn't sure about the knex/bookshelf boolean mapping. But if you think true also works, then it should be fine 😊
```suggestion
            active: true
```

Comment (rishabhgrg) [reply ghost/payments/lib/payments.js:218]: Yea think we use `true` usually but even i am confused tbh which is right/ideal. If `active:true` works locally then lets go with that as that is a better representation

**Review by rshbhgrg — APPROVED** [review]
LGTM, just a minor comment about checking/changing the `active` to `true` :)

**Pull request merged**